tests/compute/test_transform.py

```diff
@@ -594,22 +615,22 @@ def test_cast():
     assert F.array_equal(g2dst, gdst)
 
 if __name__ == '__main__':
-    test_line_graph()
-    test_no_backtracking()
-    test_reverse()
-    test_reverse_shared_frames()
-    test_simple_graph()
-    test_bidirected_graph()
-    test_khop_adj()
-    test_khop_graph()
-    test_laplacian_lambda_max()
-    test_remove_self_loop()
-    test_add_self_loop()
-    test_partition_with_halo()
-    test_metis_partition()
-    test_compact()
-    test_to_simple()
-    test_in_subgraph()
-    test_out_subgraph()
-    test_to_block()
-    test_remove_edges()
+    # test_line_graph()
+    # test_no_backtracking()
+    # test_reverse()
+    # test_reverse_shared_frames()
+    # test_simple_graph()
+    # test_bidirected_graph()
+    # test_khop_adj()
+    # test_khop_graph()
+    # test_laplacian_lambda_max()
+    # test_remove_self_loop()
+    # test_add_self_loop()
+    # test_partition_with_halo()
+    # test_metis_partition()
+    # test_compact()
+    # test_to_simple()
+    # test_in_subgraph("int32")
+    # test_out_subgraph()
+    test_to_block("int32")
+    # test_remove_edges()
```

Comment:
Why disabling these tests?

Reply:
It's s not disabled. Here is just for debug tests